### ISSUE TITLE
New build and/or 'the noobie develop setup guide', please

### ISSUE THREAD

**Issue by serhiy1994** [OpenHogwarts/hogwarts#80]
Hello mates!
I'm a big HP fan and want to contribute.
Because I'm a C# developer, I don't have problems with the programming itself. Also I worked with Unity in 2016 and still remember some workflows from those days.
But now, when I tried to open/launch the project in Unity 2020.3.f1, I got stuck.
Shortly, I'd be very glad if you:

a) help me to resolve some errors which happened when I try to launch the project. E.g., Unity asks me to fill up the PhotonProjectSettings file, otherwise it'd not be able to connect. I don't know what to write in the Settings file :(. Is it possible to use localhost or smth like?
In general case, I'm thinking about smth like youtube tutorial named like 'The guide from downloading to appearing the Hogwarts scene in the game mode'.

b) or, which is simpler, make a new release build please, so I'll be able to play instantly at least. Furthermore, I think that the new build is strongly needed to be released because the current game content and possibilities are very different from the old 1.0 release.

Anyway, any help/advice is greatly appreciated!
Greetings from Ukraine. Sorry for my English.

**Comment by tetreum** [comment]
If you opened the project with a newer version, the config may be lost, but it's always available at https://github.com/OpenHogwarts/hogwarts/blob/master/Assets/Photon%20Unity%20Networking/Resources/PhotonServerSettings.asset

I don't think a newer version is required, because there are almost no changes since 1.0.

You will have a better experience playing https://github.com/tetreum/ghosts-of-hogwarts  altougth no binary version is available. Just download the project and play it from unity.

**Comment by serhiy1994** [comment]
> If you opened the project with a newer version, the config may be lost, but it's always available at https://github.com/OpenHogwarts/hogwarts/blob/master/Assets/Photon%20Unity%20Networking/Resources/PhotonServerSettings.asset
> 
Downloaded and replaced the settings aset manually. Looks good so far, thanks.

Also I have some other error types in the Console.
a) "Rebuilding Library because the asset database could not be found!"
b) Assets/Models/<Model_name>.c4d: Unity no longer supports importing Cinema 4D files directly, but we're working with Maxon to make the experience better. To import Cinema 4D files, install Maxon's external importer. Please note that FBX files exported from Cinema4D are still supported.
c) Unsupported Assets/Models/Textures/21ac9ba6.dds file.

What should I do? Must I install Maxon? What database do the Console mean?

**Comment by tetreum** [comment]
Those are warnings, no errors. Just ignore them.

**Comment by serhiy1994** [comment]
> Those are warnings, not errors. Just ignore them.
Ok, thanks.
Edit: edited the first paragraph in my previous message (about Photon) before you replied, may you miss the editing.

**Comment by tetreum** [comment]
> Downloaded and replaced the settings aset manually. Sadly, Unity is still forces me to Run the PUN Wizard after I click Play. Must I register on the Photon Cloud etc. before playing?

nope. All required data is there. Anything that it may ask, you should be able to copy it from the linked file if it's not already autofilled.

**Comment by serhiy1994** [comment]
> > Downloaded and replaced the settings aset manually. Sadly, Unity is still forces me to Run the PUN Wizard after I click Play. Must I register on the Photon Cloud etc. before playing?
> 
> nope. All required data is there. Anything that it may ask, you should be able to copy it from the linked file if it's not already autofilled.

ok, i'll try. thanks.
